denizen_scripts/survival/repo_link/MobScripts/HuskStuff.dsc

```diff
@@ -3,7 +3,8 @@ Husk_Immolation:
   debug: false
   script:
     - repeat 15:
-      - playeffect effect:FLAME at:<context.entity.location.add[0,0.5,0]> quantity:2 offset:0.3
-      - playeffect effect:FLAME at:<context.entity.location.add[0,1.5,0]> quantity:2 offset:0.3
-      - playeffect effect:FLAME at:<context.entity.location.add[0,1.0,0]> quantity:2 offset:0.3
-      - wait 4t
+      if <server.entity_is_spawned[<context.entity.uuid>]>:
```

Comment:
missing `-` here
also, you'll want to pre-define this UUID at the beginning of the script if you want to verify with `<server.entity_is_spawned[<context.entity.uuid>]>`, elsewise you'd want to use `- if <context.entity.is_spawned||false>`

Reply:
Its being called by an entity already. Would that not define it?

Reply:
each loop will result in the script trying to read the context entity, and then pull the `.uuid` from the entitytag - this would return invalid if the entity was not valid

Reply:
Im not really able to process what <context.entity.is_spawned||false> would do for us.

I changed it to check before the loop and at the start of each iteration of the loop